topology.go

```diff
@@ -11,6 +11,8 @@ import (
 var (
 	ErrReplicasetExists    = fmt.Errorf("replicaset already exists")
 	ErrReplicasetNotExists = fmt.Errorf("replicaset not exists")
+
+	errConcurrentTopologyChangeDetected = fmt.Errorf("concurrent topology change detected")
```

Comment:
We prefer alwasy to use public errors for errors.Is and errors.As operations

Reply:
Done, I made this error public